okapi/core.php

```diff
@@ -1902,6 +1904,26 @@ public static function fix_oc_html($html)
 
         return $html;
     }
+
+    function php_ini_get_bytes($setting)
+    {
+        $setting = ini_get($setting);
+        $setting = str_replace('K', '*1024', $setting);
+        $setting = str_replace('M', '*1024*1024', $setting);
+        $setting = eval('return '.$setting.';');
```

Comment:
It is safe as long as noone with root rights enters something evil in php.ini. If someone does that, we probably have larger problems than this :)  but I like regex and have inserted one which verifies the setting. This also protects from requesting a non-numeric variable here.

The alternative to eval() would be extracting the number and (optional) multiplier and do the maths in direct PHP code.